tests/functional.js

```diff
@@ -15,12 +15,16 @@ define([
   './functional/sync_v2_reset_password',
   './functional/sync_v2_force_auth',
   './functional/sync_v3_sign_up',
+  './functional/sync_v3_sign_in',
```

Comment:
Could you add `sync_v3` sign in to `circle`'s functional tests. No need to add others, this is the most important one. Other tests will be run on teamcity anyway.